nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardFlowSynchronizer.java

```diff
@@ -1197,9 +1197,15 @@ private void updateProcessGroup(final ProcessGroup group, final ProcessGroupDTO
             }
         }
 
-        group.setDefaultFlowFileExpiration(defaultFlowFileExpiration);
-        group.setDefaultBackPressureObjectThreshold(defaultBackPressureObjectThreshold);
-        group.setDefaultBackPressureDataSizeThreshold(defaultBackPressureDataSizeThreshold);
+        if (defaultFlowFileExpiration == null) {
```

Comment:
Shouldn't these be `!= null` ? We only want to set the values if they're not null, they'll be null in the REST call when you move the component

Reply:
Yes, they should. I was moving too quickly and using the above flowFileOutboundPolicyName as an example.. but that is just setting defaults in case of null. 
Fixed.